src/debugletapi.js

```diff
@@ -225,12 +226,13 @@ DebugletApi.prototype.register_ = function(errorMessage, callback) {
 DebugletApi.prototype.listBreakpoints = function(callback) {
   var that = this;
   assert(that.debuggeeId_, 'should register first');
-  var url = API + '/debuggees/' + encodeURIComponent(that.debuggeeId_) +
-      '/breakpoints';
+  var query = { success_on_timeout: true };
   if (that.nextWaitToken_) {
-    url += '?waitToken=' + encodeURIComponent(that.nextWaitToken_);
+    query.waitToken = that.nextWaitToken;
   }
-  url += '?success_on_timeout=' + encodeURIComponent(true);
+
+  var url = API + '/debuggees/' + encodeURIComponent(that.debuggeeId_) +
```

Comment:
The e2e test world have been exercising this path. My guess is that the 'success_on_timeout' flag would have gotten dropped by the remote service – this isn't fatal – it would have meant that we would get 404 on timeouts (which would get correctly dealt with), but anyone who was monitoring the error rate of their apps would have seen unnecessary signal.